### ISSUE TITLE
No such file ..\src\nslog_win.cc for node_modules\nslog\build\nslog.vcxproj

### ISSUE THREAD

**Issue by wojohowitz** [electronjs/electron-rebuild#32]
I'm at my wits end. Long night of trying Microsoft to work. First I had to manually find cl.exe in one of my Visual Studio installs and now this:
c:\users\wojtek\documents\workspace\node\tryble\node_modules\electron-rebuild\node_modules\nslog\src\nslog.h(4): fatal error C1083: Cannot open include file: 'string': No such file or directory (..\src\main.cc) [C:\Users\Wojtek\Document
s\workspace\node\tryble\node_modules\electron-rebuild\node_modules\nslog\build\nslog.vcxproj]
c:\users\wojtek\documents\workspace\node\tryble\node_modules\electron-rebuild\node_modules\nslog\src\nslog.h(4): fatal error C1083: Cannot open include file: 'string': No such file or directory (..\src\nslog_win.cc) [C:\Users\Wojtek\Doc
uments\workspace\node\tryble\node_modules\electron-rebuild\node_modules\nslog\build\nslog.vcxproj]

God, I hate Microsoft, I'm sure you do too. Any thoughts on this one? I swear I tried everything under the sun here. I cannot install electron-rebuild to save my life. 

As an aside: Is there really no easier way? Isn't there an io.js or node version that is the same as electron? Am I missing something obvious? I can't seem to find any local (disk, not IndexDB) storage that is not native (sqlite and now trying leveldb).

**Comment by wojohowitz** [comment]
sorry, I guess I should note:
node --version
v4.2.1
npm --version
2.14.7

I was running npm 3 and thinking i'm missing peer dependencies unknown to me, i reverted back to current npm.

**Comment by paulcbetts** [comment]
> Am I missing something obvious? I can't seem to find any local (disk, not IndexDB) storage that is not native (sqlite and now trying leveldb).

How about Local Storage?

**Comment by paulcbetts** [comment]
It sounds like your VS install is messed up - do you have VS2013 Community Edition (Pro is fine too if you're a Windows dev) and have installed all the updates?

**Comment by wojohowitz** [comment]
Hmmm. Odd for me to consider Local Storage, I guess. Its a key value store, I suppose and persistent. I don't know, I guess I shouldn't think of it in the same terms as a web app. IndexDB or pouchdb then as well.

Yeah, I am certain that the VS installs are shot. Seems to happen when having different versions of VS installed and then uninstalled. Too many artifacts left behind I suppose. I thought, in the least that my PATH environment variable was clean though. 

I will look into IndexDB or puch but I do need to clean this crap up for any native module compilations.

On a side note, I cloned the nslog repo into my node_modules and things worked until the point that rebuild wanted to rebuild nslog :).

Thank you for your time though. I did really think that there was an issue with the nslog dependency  due to: No such file or directory (..\src\main.cc) .
